instrumentation/redisson-3.0/src/test/groovy/RedissonClientTest.groovy

```diff
@@ -276,10 +279,7 @@ class RedissonClientTest extends AgentTestRunner {
     assertTraces(2) {
       trace(0, 1) {
         span(0) {
-          def lockScript = "if (redis.call('exists', KEYS[1]) == 0) then redis.call('hset', KEYS[1], ARGV[2], 1);" +
-            " redis.call('pexpire', KEYS[1], ARGV[1]); return nil; end;" +
-            " if (redis.call('hexists', KEYS[1], ARGV[2]) == 1) then redis.call('hincrby', KEYS[1], ARGV[2], 1);" +
-            " redis.call('pexpire', KEYS[1], ARGV[1]); return nil; end; return redis.call('pttl', KEYS[1]);"
+          def lockScriptPattern = compile("^" + quote("EVAL ") + ".*" + quote(" 1 lock ? ?") + "\$")
```

Comment:
This magic would benefit from some comment

Reply:
Done